### PR TITLE
{vis}[foss/2018b] VirtualGL v2.6.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/v/VirtualGL/VirtualGL-2.6.1-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+hardware acceleration. Some remote display software cannot be used with OpenGL 
+applications at all. Other remote display software forces OpenGL applications 
+to use a slow, software-only renderer, to the detriment of performance as well 
+as compatibility. The traditional method of displaying OpenGL applications to 
+an X server on a different machine (indirect rendering) supports hardware 
+acceleration, but this approach requires that all of the OpenGL commands and 
+3D data be sent over the network to be rendered. That is not a tenable 
+proposition unless the 3D data is relatively small and static, unless the 
+network is very fast, and unless the OpenGL application is specifically tuned 
+for a remote X-Windows environment."""
```
Comment (vanzod) [inline easybuild/easyconfigs/v/VirtualGL/VirtualGL-2.6.1-foss-2018b.eb]: @sassy-crick Maybe just the first sentence is enough to describe the software?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/bd211073ac2103fa00b4ba915427f00c for a full test report.

File: easybuild/easyconfigs/v/VirtualGL/VirtualGL-2.6.1-foss-2018b.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+source_urls = ['https://github.com/VirtualGL/virtualgl/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['38d1d32efe97adaea6754f4559520454d4f0d2400964ebc59f8ac01f0146e23f']
+
+dependencies = [
+    ('libjpeg-turbo', '2.0.0'),
+    ('Mesa', '18.1.1'),
+    ('libGLU', '9.0.0'),
+]
+builddependencies = [('CMake', '3.12.1')]
```
Comment (vanzod) [inline easybuild/easyconfigs/v/VirtualGL/VirtualGL-2.6.1-foss-2018b.eb]: Can you please move `builddependencies` before `dependencies` for style consistency?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/38836d38b4f37136d798012ae689b43c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2606.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5df95992e51ca7e301953f6c20c1440a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2016.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/38af72c586805e53735744162b1dd135 for a full test report.

File: easybuild/easyconfigs/v/VirtualGL/VirtualGL-2.6.1-foss-2018b.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/v/VirtualGL/VirtualGL-2.6.1-foss-2018b.eb]: No problems. I shortened it to the first scentence.

File: easybuild/easyconfigs/v/VirtualGL/VirtualGL-2.6.1-foss-2018b.eb (hunk 2 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/v/VirtualGL/VirtualGL-2.6.1-foss-2018b.eb]: Done.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2517.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e1d636588d13f258f2da7e057a833b9e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**